website/templates/index/quickstart.html

```diff
@@ -2,9 +2,7 @@
     <div class="container lead">
     <h2 id="quick-start" class="display-4 mt-5">Quick start</h2>
 
-    <p>System requirements for pre-built packages: Linux, x86_64 with SSE 4.2.</p>
-
-    <ul class="nav nav-tabs" id="install-tab" role="tablist">
+    <ul class="nav nav-tabs" style="font-size: 90%; margin-top: 2rem;" id="install-tab" role="tablist">
```

Comment:
```suggestion
    <ul class="nav nav-tabs mt-1 small" id="install-tab" role="tablist">
```
...and you can keep the tests then.

See https://getbootstrap.com/docs/5.0/utilities/spacing/ and https://getbootstrap.com/docs/5.0/content/typography/#inline-text-elements